scss/utilities/_text.scss

```diff
@@ -63,8 +63,19 @@
 .text-decoration-none { text-decoration: none !important; }
 
 .text-break {
-  word-break: break-word !important; // IE & < Edge 18
-  overflow-wrap: break-word !important;
+  // We wan't to use `overflow-wrap: anywhere` to avoid issues with flex containers
```

Comment:
@mdo this whole section needs grammar fixes and personally I find it too verbose